microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -46,15 +46,15 @@ Create a zip file for your extension package.  Your extension package must inclu
 <!-- ---------------------------------- -->
 #### Review manifest field values
 
-The following fields in the manifest are automatically included in your store listing details.  The fields are read-only on the **Store listings** webpage.  The store listings webpage is described later in this article.
+The following fields in the manifest file are automatically included in your store listing details.  The fields are read-only on the **Store listings** webpage (described below).
 
-Before uploading your package to Partner Center, make sure that the field values match your preferred display on the store details webpage, as follows.
+Before uploading your package to Partner Center, inspect the manifest file to make sure that the field values match your preferred display on the store details<!-- todo: what title of webpage? bold --> webpage.
```

Comment:
what is the title of the store details webpage?

Reply:
It is called Store Listings 
![image](https://github.com/user-attachments/assets/e33dced2-f32f-4131-8260-6a5b49f700e9)

Reply:
result:

The following fields in the manifest file are automatically included in your store listing details.
The fields are read-only on the **Store listings** webpage, which is described below.

Before uploading your package to Partner Center, inspect the manifest file to make sure that 
the field values match your preferred display on the **Store listings** webpage, as follows: